lib/components/index.ts

```diff
@@ -3,3 +3,13 @@ export { WebFonts } from "./WebFonts";
 export { TextInfoTooltip, useTooltipSingleton } from "./TextInfoTooltip";
 export { GithubIcon } from "./Icons/GithubIcon";
 export { TwitterIcon } from "./Icons/TwitterIcon";
+export { PageWrap } from "./Layout/PageWrap";
+export { HeaderDesktop } from "./Header/HeaderDesktop";
+export { HeaderMobile } from "./Header/HeaderMobile";
+export { Footer } from "./Footer/Footer";
```

Comment:
What's your opinion on `index.tsx` files? I know they are controversial. I don't really care either way but I guess we should pick a pattern to use everywhere

Reply:
Not a super fan of index.tsx but not a big thing either. 
Will stick to that index pattern as this is already started like this
Changed.